src/components/fields/derived.js

```diff
@@ -461,15 +461,63 @@ function computeAxesRefOptions(axes, propsAttr) {
   return options;
 }
 
+export const TextPosition = connectToContainer(UnconnectedDropdown, {
+  modifyPlotProps: (props, context, plotProps) => {
+    const {localize: _} = props;
+    let options = [
+      {label: _('Top Left'), value: 'top left'},
+      {label: _('Top Center'), value: 'top center'},
+      {label: _('Top Right'), value: 'top right'},
+      {label: _('Middle Left'), value: 'middle left'},
+      {label: _('Middle Center'), value: 'middle center'},
+      {label: _('Middle Right'), value: 'middle right'},
+      {label: _('Bottom Left'), value: 'bottom left'},
+      {label: _('Bottom Center'), value: 'bottom center'},
+      {label: _('Bottom Right'), value: 'bottom right'},
```

Comment:
Can't all these options come from the Plotly.js schema?

Reply:
The values can, but not the translations.

Reply:
I don't I've seen us use Plotly schema for options anywhere, 
Maybe it was discussed but not implemented yet :) could you give me an example?

Reply:
TraceSelector and Marker symbols etc did this. But @alexcjohnson is correct the translations need to be statically defined somewhere. sigh.

Reply:
@alexcjohnson it's possible that we could move the static label definitions ("defaultlabel"?) into the Schema. But that would be a fairly large undertaking. It could have some nice benefits like translated tooltips. 

Out of curiousity is that something that could be supported eventually?

Reply:
Possible, but I'm not sure it's a good idea. Currently the schema serves as the reference for what the options are, but doesn't have anything to say about how those options are displayed in the editor. If we start moving some of the display info to the schema a) it'll be hard to know what goes where, b) it will be awkward to figure out who's responsible for translation (and when this should occur), and c) the editor will have to wait for plotly.js if it wants to make a small display change - or perhaps the context changes a little so the grammar used by plotly.js isn't quite right... point is this display information is only used in the editor so seems like it would be better defined in the editor too.